src/ui/table.js

```diff
@@ -30,11 +30,19 @@ export const table = () => {
       cell.setAttribute("id", `table-cell-${k}`);
       //cell.setAttribute("id", "cellTest");
       const cellText = document.createTextNode(tableArr[k]);
+
       cell.appendChild(cellText);
       if (redArray.includes(k)) {
         cell.className = "table_cell_red";
+
+        cell.addEventListener("click", function () {
+          console.log(`${tableArr[k]} red is clicked`);
+        });
       } else {
         cell.className = "table_cell_black";
+        cell.addEventListener("click", function () {
+          console.log(`${tableArr[k]} black is clicked`);
+        });
```

Comment:
instead of this, you can utilize the event object existing upon any event e.g. 
```
      cell.addEventListener("click", (event) => {
        console.log(`${event.target.innerText} ${event.target.className}`);
      });
```
and parse it, if needed.

Reply:
Ok..I did not know about target method